### PR TITLE
Add react router

### PR DESCRIPTION
**Pull request by diorshelton**
- Added react router to project

- Add routes for  Home and  Login

### CODE REVIEW

File: src/modules/home/routes/HomeRoute.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { Home } from "../components/Home";
+
+export const homeRoute = {
+	path: "/",
+	element: <Home />,
+};
```
Comment (Leigh-K) [inline src/modules/home/routes/HomeRoute.js]: One thing I shouldve mentioned Id like is path to be saved into a const. I think later we will use it for navigation
```
const homePath = '/'

export const homeRoute = {
	path: "homePath",
	element: <Home />,
};
```

Comment (Leigh-K) [reply src/modules/home/routes/HomeRoute.js]: So in future if we wanna route there from a function we can export it and route

File: src/modules/login/Routes/LoginRoute.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { Login } from "../components/Login";
+
+export const loginRoute = {
+	path: "/login",
+	element: <Login />,
+};
```
Comment (Leigh-K) [inline src/modules/login/Routes/LoginRoute.js]: same comment

```
const loginPath = '/login'

export const loginRoute = {
	path: "loginPath",
	element: <Login />,
};
```

**Review by Leigh-K — CHANGES_REQUESTED** [review]
You need to run prettier as you can see the indentation is off

**Comment by Leigh-K** [comment]
the login route folder is still `Route` change to lowercase `routes`

**Comment by diorshelton** [comment]
Prettier automatically runs when I save. Could you be specific about the indentation that needs to be corrected?

**Comment by Leigh-K** [comment]
> Prettier automatically runs when I save. Could you be specific about the indentation that needs to be corrected?

Yep your prettier settings are not standard. 

heres my suggestion on how to fix. 

May as well take the prettier take and make it as part of this PR. 

Easy steps to add prettier./ 

from CLI in project folder

```
npm install --save-dev --save-exact prettier
 ```
 
 Prettier is now installed. 
 
 Add a file called 
 `.prettierrc.json`
 in root directory.
 
 add this into that file
 ```
 {
    "trailingComma": "es5",
    "tabWidth": 4,
    "semi": false,
    "singleQuote": true
}
```

Add in script into the package.json
```
"prettier": "prettier --write ./src"
```
Sop you can see our other scripts start, build. test, eject. This would be like that, so then you can go to your command line and run 
`npm run prettier` and it will correctly format every file. 
It should now also mean your vscode uses the correct settings when you save too.

**Pull request closed without merging**